Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN conda install -c conda-forge google-cloud-sdk && \
     rm kubectl && \
     # Install uv and skypilot
     curl -LsSf https://astral.sh/uv/install.sh | sh && \
-    ~/.local/bin/uv pip install --prerelease allow 'azure-cli>=2.65.0' 'omegaconf>=2.4.0dev3' --system && \
+    ~/.local/bin/uv pip install --prerelease allow 'azure-cli>=2.65.0' --system && \
```

Comment:
```suggestion
    ~/.local/bin/uv pip install 'azure-cli>=2.65.0' --system && \
```

Reply:
Also, thanks so much @SeungjinYang for addressing this issue so quickly, it's much appreciated!!! :star_struck: